core/storage/app_feedback_report/gae_models.py

```diff
@@ -147,7 +147,7 @@ def create(
         Args:
             platform: str. The platform the report is submitted on.
             submitted_on: datetime.datetime. The date and time the report was
-                submitted, in UTC.
+                submitted, in the user's local time zone.
```

Comment:
How does datetime store the timezone?

Reply:
Let me know if I'm misunderstanding this but datetime.datetime has a [tzinfo](https://docs.python.org/3/library/datetime.html#datetime.datetime.tzinfo) property from which we can add a [timezone](https://docs.python.org/3/library/datetime.html#datetime.timezone) object that specifies the offset from UTC. If we can use this here then we only need to add timezone to the Android report and then the server can process it to a correct local time once it's received in the backend.

Reply:
That seems reasonable--thanks for clarifying!